crates/astria-sequencer-relayer/local.env.example

```diff
@@ -19,10 +19,14 @@ ASTRIA_SEQUENCER_RELAYER_PRETTY_PRINT=false
 # `ASTRIA_SEQUENCER_RELAYER_PRETTY_PRINT` is set to `true`.
 NO_COLOR=
 
-# Address of sequencer/cometbft/tendermint to request new blocks.
+# Address of cometbft/tendermint to request new block heights.
 # 127.0.0.1:26657 is the default socket address at which cometbft
 # serves RPCs.
-ASTRIA_SEQUENCER_RELAYER_SEQUENCER_ENDPOINT="http://127.0.0.1:26657"
+ASTRIA_SEQUENCER_RELAYER_COMETBFT_ENDPOINT="http://127.0.0.1:26657"
+
+# Address of sequencer applcation gRPC service for requesting new blocks.
+# 127.0.0.1:8080 is the default socket address for its gRPC server.
+ASTRIA_SEQUENCER_RELAYER_SEQUENCER_ENDPOINT="http://127.0.0.1:8080"
```

Comment:
Can we call this like `ASTRIA_SEQUENCER_RELAYER_SEQUENCER_GRPC_ENDPOINT`? Avoid name collision with previous config when different expected value

Reply:
done